x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/exploration_results_table/use_exploration_results.ts

```diff
@@ -107,6 +117,38 @@ export const useExplorationResults = (
     jobConfig?.dest.index,
     JSON.stringify([searchQuery, dataGrid.visibleColumns]),
   ]);
+  const predictionFieldName = jobConfig ? getPredictionFieldName(jobConfig.analysis) : undefined;
```

Comment:
Could this return the default prediction name instead of `undefined`?

Reply:
Updated in https://github.com/elastic/kibana/pull/73561/commits/ef815d11a2bf334d703366daff3519b38b848e11